.github/workflows/cron-licenses.yml

```diff
@@ -6,11 +6,10 @@ on:
     - cron: "7 0 * * 1"
```

Comment:
Are you able to add a conditional to the cron's so that they only run when the repo is "go-gitea/gitea"?

Reply:
Not that familiar with this topic, but I can search docs.

Reply:
https://github.com/go-gitea/gitea/pull/24620/commits/ce3cf6b6c5bd7eb9c3af4e381012be2aa8dbd9b4 should fix this.